storage/interface.go

```diff
@@ -44,8 +46,14 @@ type SampleAndChunkQueryable interface {
 	ChunkQueryable
 }
 
+// ExemplarAppendable allows creating exemplar appenders.
+type ExemplarAppendable interface {
+	// Appender returns a new exemplar appender for the exemplar storage.
+	ExemplarAppender() ExemplarAppender
+}
```

Comment:
Correct me if I'm wrong, but this interface doesn't look like its used in any meaningful way?  I mean, the scrape code uses `Appender()` (which returns something that implements `ExemplarAppender`).  The tests use it against the exemplar store, but that store also implements `ExemplarAppender` IFAICT.

So, just remove this?

Reply:
Yep, removing. I'm still questioning the state the interfaces are in but I think at this point we can just move forward with the rest of them as is until we run into a need to change them.